components/browser/menu/src/test/java/mozilla/components/browser/menu/BrowserMenuBuilderTest.kt

```diff
@@ -28,7 +28,7 @@ class BrowserMenuBuilderTest {
         val recyclerView: RecyclerView = popup.contentView.findViewById(R.id.mozac_browser_menu_recyclerView)
         assertNotNull(recyclerView)
 
-        val recyclerAdapter = recyclerView.adapter
+        val recyclerAdapter = recyclerView.adapter!!
```

Comment:
What's the difference to change it **!!**? Just to be more explicit about nullability?

Reply:
They added @Nullable in the latest version of the support library. Now we need to deal with that :)

Reply:
👍